ClLike/cl_like/cl_like.py

```diff
@@ -87,13 +87,14 @@ def get_lmax_from_kmax(cosmo, kmax, zmid):
             # Give galaxy clustering an ell_max
             if t.quantity == 'galaxy_density':
                 # Get lmax from kmax for galaxy clustering
-                if 'kmax' in self.defaults[b['name']]:
-                    kmax = self.defaults[b['name']]['kmax']
-                else:
-                    kmax = kmax_default
-                lmax = get_lmax_from_kmax(cosmo_lcdm,
-                                          kmax, zmid)
-                self.defaults[b['name']]['lmax'] = lmax
+                if 'lmax' not in self.defaults[b['name']]:
+                    if 'kmax' in self.defaults[b['name']]:
+                        kmax = self.defaults[b['name']]['kmax']
+                    else:
+                        kmax = kmax_default
+                    lmax = get_lmax_from_kmax(cosmo_lcdm,
+                                              kmax, zmid)
+                    self.defaults[b['name']]['lmax'] = lmax
```

Comment:
`else`?

Reply:
else  "self.defaults[b['name']]['lmax']" already exists and has some value